src/pyocr/builders.py

```diff
@@ -284,6 +341,29 @@ def __str__():
         return "Raw text"
 
 
+class DigitBuilder(TextBuilder):
+    """
+    If passed to image_to_string(), image_to_string() will return a simple
+    string of digits. 
+    This string will be the output of the OCR tool, as-is. 
+    In other words, the raw text as produced by the tool when the input is
+    assumed to be [0-9.] only. Note that depending on the tool, 
+    some other characters may be included.
```

Comment:
My bad. In my first comment, I told you to write that. However, in your implementation, you actually raise NotImplementedError(), which is fine too, but should be documented here.